src/microbe_stage/ProcessSystem.cs

```diff
@@ -64,35 +64,34 @@ public ProcessSystem(Node worldRoot)
 
```

Comment:
The above two lines should be removed
```
        // Compute all process efficiencies once
        var efficiencies = ComputeOrganelleProcessEfficiencies(enumerated, biome);
```
because after this change they shouldn't be required.